renovate.json

```diff
@@ -9,6 +9,15 @@
     "fileMatch": ["requirements-test.txt", "requirements-composer.txt", "constraints.txt", "constraints-test.txt"]
   },
   "packageRules": [
+    {
+      "packagePatterns": ["pytest"],
+      "separateMinorPatch": true
+    },
+    {
+      "packagePatterns": ["pytest"],
+      "updateTypes": ["patch"],
+      "enabled": false
+    },
     {
       "packagePatterns": ["pytest"],
       "updateTypes": ["minor", "major"]
```

Comment:
DO we need to also remove this section? Or do these rules work with the existing pattern?

Reply:
nvm, i see the comment